src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java

```diff
@@ -107,7 +107,8 @@
 
     /**
      * If specified, this parameter will cause the dependency tree to be written using the specified format. Currently
-     * supported format are: <code>text</code> (default), <code>dot</code>, <code>graphml</code> and <code>tgf</code>.
+     * supported format are:
+     * <code>text</code> (default), <code>dot</code>, <code>graphml</code>, <code>tgf</code> and <code>xml</code>.
```

Comment:
let's reformat this to an unordered list

Reply:
Sorry, but i do not understand the request

Reply:
Like

    <ul>
      <li>text (default)</li>
      <li>dot</li>
      ..

Or 

    <dd>
      <dt>text</dt>
      <dd>A textual tree (default)<dd/>
      <dt>dot</dt>
      <dd>Textual output based on the DOT graph description language</dd>
    ..